atomic_reactor/build.py

```diff
@@ -75,18 +76,24 @@ def __init__(self, logs=None, fail_reason=None, image_id=None,
                        should be set as labels on OpenShift build
         :param skip_layer_squash: boolean, direct post-build plugins not
                                   to squash image layers for this build
+        :param oci_image_path: str, path to OCI image directory
         """
         assert fail_reason is None or bool(fail_reason), \
             "If fail_reason provided, can't be falsy"
         # must provide one, not both
-        assert bool(fail_reason) != bool(image_id), \
+        assert not (fail_reason and image_id), \
```

Comment:
assert not (fail_reason and (image_id or image_id)

instead having 2 checks?

Reply:
I wanted to have different error msgs